Xamarin.Forms.Core/MergedStyle.cs

```diff
@@ -5,163 +5,160 @@
 
 namespace Xamarin.Forms
 {
-	public partial class VisualElement
+	sealed class MergedStyle : IStyle
```

Comment:
All I did here was remove the public partial class VisualElement line.

Didn't make any formatting changes, Github just thinks its all different.